frontend/src/app/modules/authentication/authentication-routing.module.ts

```diff
@@ -9,6 +10,8 @@ const routes: Routes = [
         component: AuthenticationPageComponent,
         children: [
             { path: 'registration', component: RegistrationComponent },
+            { path: 'login', component: LoginComponent },
+            { path: '**', component: LoginComponent },
```

Comment:
either redirect or set in all to /

Reply:
actually i didn't really understand, redirect to page not found or just set / when wrong route?

Reply:
Redirect to /login)
Or set LoginComponent to /

Reply:
got it, and implement this